### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by muhiddinsgithub**
Can you check and permit to merge to master?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,649 @@
...
+
+```text
+/api
+.
+├── /auth
+│   └── POST
+│       └── /login
+├── /users
+│   └── GET
+│       ├── /:user_id
```
Comment (malcolmkiano) [inline README.md:34]: we changed this to username

File: README.md [hunk 2]
```diff
@@ -1,35 +1,649 @@
...
+  ?
+}
+
+// res.body
+
+[
+  {
+    id: String,
+    name: String,
+    creator: String,
```
Comment (malcolmkiano) [inline README.md:188]: this field was removed, and replaced with `isOwner` which is a Boolean

File: README.md [hunk 3]
```diff
@@ -1,35 +1,649 @@
...
+
+### GET `/api/projects/user/`
+
+```js
+// req.user.id
+
+{
+  user_id: Number;
+}
+
```
Comment (malcolmkiano) [inline README.md:207]: this endpoint doesn't require any request body

File: README.md [hunk 4]
```diff
@@ -1,35 +1,649 @@
...
+// res.body
+
+[
+  {
+    id: String,
```
Comment (malcolmkiano) [inline README.md:212]: `id` is no longer included in the response

File: README.md [hunk 5]
```diff
@@ -1,35 +1,649 @@
...
+  trello_url: String,
+  github_url: String
+}
+
+// res.body
+
+  {
+    id: String,
+    name: String,
+    creator: String,
```
Comment (malcolmkiano) [inline README.md:268]: field was replaced with `isOwner` which is a Boolean

**Comment by muhiddinsgithub** [comment]
Sure

**Pull request merged**